exercises/K-functions-parameters/exercise4.js

```diff
@@ -1,5 +1,8 @@
 // Declare your function first
 
 // Call the function and assign to a variable `sum`
-
+function summed(a,b){
+    return a+b
+};
+let sum =summed(13,24)
```

Comment:
`summed` is a cool variable name :sunglasses:. Can you explain why it is a _good_ variable name in this context though?